test/router/v2/SynapseRouterV2.Inspection.t.sol

```diff
@@ -284,7 +284,8 @@ contract SynapseRouterV2InspectionTest is BasicSynapseRouterV2Test {
             rawParams: getSwapParams(address(poolNethWeth), 1, 0)
         });
 
-        SwapQuery[] memory actualQueries = router.getOriginAmountOut(tokenIn, tokenSymbols, amountIn);
+        SwapQuery[] memory actualQueries = new SwapQuery[](1);
+        actualQueries[0] = router.getOriginAmountOut(tokenIn, tokenSymbols[0], amountIn);
 
         assertEq(expectedQueries.length, actualQueries.length);
         checkSwapQueries(expectedQueries, actualQueries);
```

Comment:
Small nit: will we ever need to compare arrays of `SwapQuery` structs with the change introduced in the PR?

Reply:
Likely not. This is simply me trying to minimize changes in the PR

Reply:
@ChiTimesChi should be good to go now.